crates/air_formatter_test/src/spec.rs

```diff
@@ -28,6 +28,9 @@ impl<'a> SpecTestFile<'a> {
 
         let input_code = std::fs::read_to_string(input_file).unwrap();
 
+        // Normalize to Unix line endings
+        let input_code = line_ending::normalize(input_code);
+
```

Comment:
We don't have a way to set options on a per test basis right now, but eventually it will be nice to have a test where we force the format output to be CRLF to ensure it is working right